### ISSUE TITLE
install newest version through git

### ISSUE THREAD

**Issue by tdg2088** [rtavenar/tslearn#83]
I'm using tslearn v0.1.19.2 on windows 10 64-bit and python 3.6.7  well,but when I trying to use latest github-hosted version via command :pip install git+https://github.com/rtavenar/tslearn.git ,I got an error:
running build_ext
error: [WinError 2] 系统找不到指定的文件
I think it due to wrong installation or wrong config of Cython,
would you upload a latest windows 10 64-bit  version to conda-forge please?

**Comment by rtavenar** [comment]
Hi @tdg2088 

Have you tried `pip install Cython` before running the command you specified in your message? What does it give?

Best,
Romain

PS: I do not really understand what is the event that triggers a new upload to conda-forge, @rth could you help on this?

**Comment by rth** [comment]
> PS: I do not really understand what is the event that triggers a new upload to conda-forge, @rth could you help on this? Is this manual?

Normally it would be necessary to make a PR and increment verion in https://github.com/conda-forge/tslearn-feedstock

Now however there is a bot that makes those PRs automatically, so often it's only necessary to merge them.
Merged https://github.com/conda-forge/tslearn-feedstock/pull/19 so 0.1.23 should be available on conda-forge in a few hours.

**Comment by tdg2088** [comment]
I solved it buy uninstall all things about anaconda and visual studio on my windows system,and than I install Visual Studio Community 2017 (include python 3.6 , visual C++ v14 and visual C++ v17 ) through Visual Studio Installer ,than  I can build tslearn v0.1.23 success on my computer.
@rtavenar I have tried to run  pip install Cython first,but I got the same error, I think it due to wrong config of Cython setting.
thanks for both  @rtavenar  and @rth , you did a great help for me.
